docs/configuration/cluster_manager/cds.rst

```diff
@@ -0,0 +1,64 @@
+.. _config_cluster_manager_cds:
+
+Cluster discovery service
+=========================
+
+The cluster discovery service (CDS) is an optional API that Envoy will call to dynamically fetch
+cluster manager members. Envoy will reconcile the API response and add, modify, or remove known
+clusters depending on what is required.
+
+.. code-block:: json
+
+  {
+    "cluster": "{...}",
```

Comment:
for better or worse this is the standard we have been using. I agree at this point just using {} would probably be better. I can do a bulk find/replace in a follow up change.